packages/tasit-action/src/DecentralandMana.js

```diff
@@ -0,0 +1,11 @@
+import Contract from "./Contract";
+import manaTokenABI from "./abi/MANAToken.json";
+
+export default class DecentralandMana extends Contract {
+  #contract;
+
+  constructor(address, wallet) {
```

Comment:
I suppose we could consider DRYing this up with other classes where we do this, but I think that's probably DRY-overkill. So I'm leaning towards no change here.

Reply:
I've just realized that that `#contract` field is unnecessary.

Reply:
Yeah, true!

Reply:
Resolving this since you addressed it in a newer commit